libr/util/table.c

```diff
@@ -462,6 +462,37 @@ R_API char *r_table_tosimplestring(RTable *t) {
 	return r_strbuf_drain (sb);
 }
 
+R_API char *r_table_tor2cmds(RTable *t) {
+	RStrBuf *sb = r_strbuf_new ("");
+	RTableRow *row;
+	RTableColumn *col;
+	RListIter *iter, *iter2;
+
+	r_strbuf_appendf (sb, ",h ");
+	r_list_foreach (t->cols, iter, col) {
+		char fmt = col->type == &r_table_type_string? 's': 'x';
+		r_strbuf_appendf (sb, "%c", fmt);
+	}
+	r_list_foreach (t->cols, iter, col) {
+		r_strbuf_appendf (sb, " %s",  col->name);
+	}
+	r_strbuf_append (sb, "\n");
+
+	r_list_foreach (t->rows, iter, row) {
+		char *item;
+		int c = 0;
+		r_strbuf_appendf (sb, ",r ");
+		r_list_foreach (row->items, iter2, item) {
+			RTableColumn *col = r_list_get_n (t->cols, c);
```

Comment:
why is this required? I think that if a row has 3 items, there are at least 3 columns in the table, otherwise something would be very wrong in the table. Am I missing anything?

Reply:
would you prefer a runtime assert?

Reply:
if we don't think this is 100% true yeah, maybe better, though I would expect this to be always true, if the table API is ok.

Reply:
See line :249 .. it is not reliable so better keep this safe if until we get proper unit and r2 tests for this